src/popup/comment.js

```diff
@@ -0,0 +1,35 @@
+export default class InvolvementApi {
+  constructor() {
+    this.url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/hWosAqiVeXtujgpmRoG5/comments?item_id=item1';
+  }
```

Comment:
At the end of the URL where it says 'item1' should be replaced with the item id, that id should be passed as a parameter in the addComments(itemId) method, and then when you use the fetch method it should be something like this `fetch('${this.url}${itemId}', { the rest of the code })` that way if you pass the id 6 as a parameter you'll get all the comments from the item 6, and for checking if it working you'll need to post some comments for the specifics ids that you'll check and remember to use postman to check if your POSTS were successful